reference/constraints/Locale.rst

```diff
@@ -90,5 +96,18 @@ This message is shown if the string is not a valid locale.
 
 .. include:: /reference/constraints/_payload-option.rst.inc
 
+canonicalize
+~~~~~~~~~~~~
+
+**type**: ``boolean`` **default**: ``false``
+
+.. versionadded:: 4.1
+    Using this option with value `false` was deprecated in Symfony 4.1 and it will
```

Comment:
In this `versionadded` directive, the `false` and `false` code have single backticks but the RST format requires double backticks